### ISSUE TITLE
Not finding airpods pro

### ISSUE THREAD

**Issue by mzaur** [Gaulomatic/AirPodsSanity#12]
This was working fine for a while, but recently it stopped picking up my Airpods Pro. It's just completely missing from the list. I tried deleting and reinstalling, and it works for a bit. But then after disconnecting, airpods are not found. 

I just restarted the program and now airpods are in the list again. I put them back in the case to disconnect, and then connect them again, no longer in the list so I can't choose Airpods. 
Not sure what to do. 

I'm on MacOS 14.5

**Comment by Gaulomatic** [comment]
The AirPods need to be connected to be detected. While disconnected, they will not be displayed in the menu.

**Comment by mzaur** [comment]
The AirPods are connected. I’m saying that they do not show up after
disconnecting and connecting again. They only connect once and I have to
restart the app to get them to connect again.

**Comment by Gaulomatic** [comment]
That sounds quite strange indeed. Do you have the same problem with other devices as well?

**Comment by mzaur** [comment]
Yes just tested with regular airpods (not pro) and same thing. Did not show up in the list. I attached a screenshot.
Is there a way to reset the app completely? it worked for a while and then this started happening. I tried reinstalling but that didn't fix it. 
<img width="329" alt="Screenshot 2024-08-12 at 12 34 11 PM" src="https://github.com/user-attachments/assets/1b762f99-ae97-4344-86f5-086e7504f0f6">
<img width="317" alt="Screenshot 2024-08-12 at 12 34 06 PM" src="https://github.com/user-attachments/assets/6086c986-517c-4140-aa5a-4dd6a851f0e5">

**Comment by Gaulomatic** [comment]
You can delete the following folder to reset the app:

~/Library/Containers/eu.punke.AirPods-Sanity

Close the app prior.

**Comment by togume** [comment]
FYI - this happened to me, too. I have AirPods Pro 2, and I've had to restart the application for it to detect them. I just deleted the Library files to test again and try to reproduce when they get lost.
